app/models/user.rb

```diff
@@ -14,7 +10,11 @@ class User < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :likes
 
+  def admin?
+    role == 'admin'
+  end
```

Comment:
- Thumbs up 👍🏿 for implementing this in the user model.

Reply:
thanks ^^